contracts/ClaimsManager.sol

```diff
@@ -464,16 +463,19 @@ contract ClaimsManager is
                 claim.policyHash,
                 claim.amountInUsd
             );
-            uint256 amountInApi3 = convertUsdToApi3(clippedAmountInUsd);
-            updateQuotaUsage(msg.sender, amountInApi3);
+            clippedAmountInApi3 = convertUsdToApi3(clippedAmountInUsd);
+            updateQuotaUsage(msg.sender, clippedAmountInApi3);
             emit ResolvedDisputeByAcceptingClaim(
                 claimIndex,
                 claim.claimant,
                 claim.beneficiary,
-                amountInApi3,
+                clippedAmountInApi3,
```

Comment:
I guess for consistency, we would also want to update the `AcceptedClaim` event to use the `clippedAmountInApi3` naming?
```solidity
    event AcceptedClaim(
        uint256 indexed claimIndex,
        address indexed claimant,
        address beneficiary,
        uint256 amountInApi3,
        address sender
    );
```
Would it not also make sense to include the `clippedAmountInUsd` in all these logs (`AcceptedClaim`,  `AcceptedSettlement`, `ResolvedDisputeByAcceptingClaim`, `ResolvedDisputeByAcceptingSettlement`)  alongside the `clippedAmountInApi3`? It would allow you to figure out if the amount was clipped or not (and by how much) by comparing it to the claim amount/settlement

Reply:
Agreed